README.md

```diff
@@ -97,6 +115,13 @@ $ git hooks shared list --with-url
 ```
 
 The install script offers to set these up for you, but you can do it any time by changing the global configuration variable. These repositories will be checked out into the `~/.githooks.shared` folder, and are updated automatically after a `post-merge` event (typically a `git pull`) on any local repositories. The layout of these shared repositories is the same as above, with the exception that the hook folders (or files) can be at the project root as well, to avoid the redundant `.githooks` folder.
+The additional global configuration parameter `githooks.failOnNotExistingSharedHooks`:
```

Comment:
How about this?

```
An additional global configuration parameter `githooks.failOnNotExistingSharedHooks` makes hooks fail with an error if any shared hook configured in `.shared` is missing, meaning `git hooks update` has not yet been called.
Note that shared hooks are automatically updated on clone.
```

Also, this reminds me, that we should have added a `git hooks config [enable|disable] fail-on-non-existing-shared-hooks` to the cli.

Reply:
Hm... yes. I would use `failOnNotExistingSharedHooks` same as in the config.

Reply:
No its good as you propsed.